src/lib.rs

```diff
@@ -140,9 +169,16 @@ impl HexView {
             cursor: Vec2::zero(),
             data: data.into_iter().map(|u| *u.borrow()).collect(),
             state: DisplayState::Disabled,
+            config: HexViewConfig::default()
         }
     }
 
+    ///Change configuration values for spacers and separators
```

Comment:
I would like to see a small example, see the other comments of this struct

Reply:
I'm not sure how to add value here without doing the same I did on the examples/config.rs file.

Reply:
Something like (please remove the space in the tripe backtick (```)

```
This function allows the customization of the `HexView` output.

For options and explanation of every possible option, see the `HexViewConfig` struct.

#Examples

`` `
# use cursive_hexview::{DisplayState,HexView};
let view = HexView::new().config(HexViewConfig {
    bytes_per_line: 8,
    ..Default::default()
});
`` `

```

Reply:
let me know if [this](https://github.com/palako/cursive_hexview/commit/935d10a3f180998a57bf694a8ba7c156a979ae4d) is good